avocado-setup.py

```diff
@@ -335,6 +348,8 @@ def bootstrap(enable_kvm=False):
                   err_str="Failed to create test repo dir. Error: ")
     for repo in TEST_REPOS:
         get_repo(repo, TEST_DIR)
+    if not (len(prescript) == 1 and not prescript[0]):
+        configure_prepostscript(prescript, 'pre')
```

Comment:
```
for script in prescript:
     configure_prepostscript(script, 'pre')
```

Reply:
ah loop I am maintaining in configure_prepostscript as in that function I am creating dir as well , let me think seems me good idea